### ISSUE TITLE
Multiclass train/validation split is made repeatedly across each class

### ISSUE THREAD

**Issue by jenspete** [Abe404/RootPainter3D#30]
The decision of whether to add images to the validation or training set is made when an annotation is saved. This however happens for each class and so images may be in the training set for one class and validation set for another class. The model could therefore potentially overfit to the validation set.

A simple fix could be to check if the file is already in the training or validation set for another class before choosing the set.

**Comment by Abe404** [comment]
Good point. I'm working on another issue at the moment but I will try to get this addressed within the next 10 days. Let me know if you need it sooner.
